### DISCUSSION TITLE
CSV Import - goodreads data [Jun 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2506] [Discord Threads]
> 🙋‍♂️ **zblesk**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### could you pls DM me the csv  
#  
> 🙋‍♂️ **zblesk**  
#### sent. Thanks.  
#  
> 💁 **o1lab**  
#### thank you - will see & get back to u  
#  
> 🙋‍♂️ **wkw**  
#### <@175168862528995328> this issue will be tracked here.    
####    
#### https://github.com/nocodb/nocodb/issues/2439  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/988444505520889876)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
